python/ray/data/context.py

```diff
@@ -14,6 +14,9 @@
 # Whether block splitting is on by default
 DEFAULT_BLOCK_SPLITTING_ENABLED = False
 
+# Whether pandas block format is enabled.
+DEFAULT_ENABLE_PANDAS_BLOCK = True
```

Comment:
Can we add a TODO to remove this feature flag once stable? (should be a temporary thing)